### DISCUSSION TITLE
3D binning of point cloud

### DISCUSSION THREAD

**Discussion by cnicho35** [isl-org/Open3D#4599] [Q&A]
Are there any functions available to bin points of the point cloud based on location? [Here is a good picture that represents the 3D bins I would like to group the points into](https://www.researchgate.net/profile/Victor-Ivanov/publication/323173163/figure/fig1/AS:593934745092096@1518616486142/Hierarchical-binning-in-3D-space.png)

stackoverflow: https://stackoverflow.com/questions/70717111/how-can-i-efficiently-bin-3d-point-cloud-data-based-on-point-coordinates-and-des

**Comment by benjaminum** [comment]
There are functions for voxelization. See this tutorial http://www.open3d.org/docs/release/tutorial/geometry/voxelization.html

**Reply by cnicho35** [reply]
Thank you, when this was written I was unfamiliar with the jargon. I'm all set!
